src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -80,11 +89,9 @@
    * batch execution, which means that they are re-initialized on every {@link #processBatch(List)}
    * call.
    */
-  private Set<CategoryDraft> existingCategoryDrafts = new HashSet<>();
-
   private Set<CategoryDraft> newCategoryDrafts = new HashSet<>();
```

Comment:
existingCategoryDrafts is removed but the Javadoc is there 😄 

also, the other 2 variables ar not used, I wonder if we need to remove newCategoryDrafts and referencesResolvedDrafts ?

Reply:
2d7ee51c7cb0b864d4ad135550f44f302601631a